ckanext/qa/reports.py

```diff
@@ -80,14 +80,14 @@ def openness_index(include_sub_organizations=False):
         row = OrderedDict((
             ('organization_title', results[org_name]['organization_title']),
             ('organization_name', org_name),
-            ('total_stars', total_stars),
-            ('average_stars', average_stars),
+            (_('total_stars'), total_stars),
+            (_('average_stars'), average_stars),
```

Comment:
why are we editing the internal key for this report?  Translation layer should only happen on visible strings not functional internals.

Reply:
Agreed, localising our internal keys like this is begging for problems down the line.

Reply:
Does this translate to csv headers? how does that occur?